istioctl/cmd/proxyconfig.go

```diff
@@ -526,34 +616,45 @@ func proxyConfig() *cobra.Command {
 	endpointConfigCmd.PersistentFlags().IntVar(&port, "port", 0, "Filter endpoints by Port field")
 	endpointConfigCmd.PersistentFlags().StringVar(&clusterName, "cluster", "", "Filter endpoints by cluster name field")
 	endpointConfigCmd.PersistentFlags().StringVar(&status, "status", "", "Filter endpoints by status field")
+	endpointConfigCmd.PersistentFlags().StringVarP(&configDumpFile, "file", "f", "",
+		"Envoy config dump JSON file")
 
 	bootstrapConfigCmd := &cobra.Command{
-		Use:   "bootstrap <pod-name[.namespace]>",
+		Use:   "bootstrap [<pod-name[.namespace]>]",
 		Short: "Retrieves bootstrap configuration for the Envoy in the specified pod",
 		Long:  `Retrieve information about bootstrap configuration for the Envoy instance in the specified pod.`,
 		Example: `  # Retrieve full bootstrap configuration for a given pod from Envoy.
   istioctl proxy-config bootstrap <pod-name[.namespace]>
```

Comment:
Should have an example like the others?

Reply:
Added an example.  (Originally it didn't have one because no one uses this subcommand as far as I can see!)